scripts/micro_benchmarks.py

```diff
@@ -238,13 +239,28 @@ def __valid_dir_path(file_path: str) -> str:
             raise ArgumentTypeError('{} does not exist.'.format(file_path))
         return file_path
 
+    def __csproj_file_path(file_path: str) -> dotnet.CSharpProjFile:
+        file_path = __valid_file_path(file_path)
+        return dotnet.CSharpProjFile(
+            file_name=file_path,
+            working_directory=path.dirname(file_path)
+        )
+
+    microbenchmarks_csproj = path.join(
+        get_repo_root_path(), 'src', 'benchmarks', 'micro',
+        'MicroBenchmarks.csproj'
+    )
     parser.add_argument(
-        '--working-directory',
-        dest='working_directory',
+        '--csproj',
```

Comment:
this change is required, but it's gonna need an extra change in the `.yml` file (because Arcade copies the `/src`  and `/scripts` to two different folders instead of just sending entire repo to `Helix`...)